pingen_env/models/res_company.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, fields, models
+
+from odoo.addons.server_environment import serv_config
+
+
+class ResCompany(models.Model):
+    _inherit = ["res.company"]
+
+    pingen_clientid = fields.Char(
+        compute="_compute_pingen_env", required=False, readonly=True
+    )
+    pingen_client_secretid = fields.Char(
+        compute="_compute_pingen_env", required=False, readonly=True
+    )
+    pingen_organization = fields.Char(
+        compute="_compute_pingen_env", required=False, readonly=True
+    )
+    pingen_staging = fields.Boolean(
+        compute="_compute_pingen_env", required=False, readonly=True
+    )
+    pingen_webhook_secret = fields.Char(
+        compute="_compute_pingen_env", required=False, readonly=True
+    )
+
+    @api.depends()
+    def _compute_pingen_env(self):
+        global_section_name = "pingen"
+        for company in self:
+            # default vals
+            config_vals = {
+                "pingen_clientid": "",
+                "pingen_client_secretid": "",
+                "pingen_organization": "",
+                "pingen_staging":False,
```

Comment:
Actually, I think we want a `True` value as default here, to avoid sending anything to prod without setting it explicitly.